### ISSUE TITLE
Unfair duplicate errors with parametric modals 

### ISSUE THREAD

**Issue by BoyvdR** [manyfold3d/manyfold#3074]
**Describe the bug**
Models that have variations or a collection of modals will gave the duplicate error after uploading them. Think of models like Gridfinity, which have different sizes in width and height, but with similair names (1x1x1, 1x1x2, 1x1x3, 1x2x1 etc) will gave a duplicate error.

**To Reproduce**
1. Go to /models/new
2. Click on _browse files_ or upload a large collection of modals which are similair name
3. Click on _Create modals_
4. See error under /problems

**Expected behavior**
No duplicate error. The files are not the same size or aren't the same file. Or let the 3d model be shown like all the none duplicate files but with a "possible duplicate" tag. 

**Screenshots**
![image](https://github.com/user-attachments/assets/417c868f-dc90-462f-95b7-69bdd1f1a933)
![image](https://github.com/user-attachments/assets/9332e71d-ab33-4ef2-be3a-63243d8f02d8)

**Comment by BoyvdR** [comment]
The issue seems to be in the uploading the files via de drag and drop interface.. So it only happens when you use the drag and drop with a large amount of files.

I use the brave browser on pop-os with the Dolphin file explorer.

**Comment by Floppy** [comment]
Oh! There's the issue! The file size for that files is shown as zero bytes (hence the load error), which will also mean that anything else with 0 length will get the same digest and show as a duplicate. Can you check that the files are *not* in fact all zero size? Rescan may help if they've been analysed wrong somehow.

I should probably add a different error for files that are zero-length, rather than them showing up as duplicates.

**Comment by BoyvdR** [comment]
Yes, that made it a little unclear to me what actually happend.. 
All the files were bigger then 0, most of them around 2mb, I think. I decided on removing en re uploading them. I can reproduce the issue and then analyze to see if the will be analyze correctly.  

That would be nice feature!  I was a bit stuck, I saw they where 0 bytes, but thought that was beceaus of some feature made sure that a model only would be taken up in the database if it isn't a duplicate.
